### ISSUE TITLE
Status?

### ISSUE THREAD

**Issue by justjacksonn** [go-raml/raml#5]
It shows about a year ago the last update. Does this fully support 0.8 spec? Is there work ongoing to support 1.0 now?

**Comment by ryanemax** [comment]
I'v imported the package, the 0.8 spec is OK. But haven't support 1.0 yet.

You should Have a try for this fork, which plan to support 1.0 first:

http://github.com/buddhamagnet/raml

**Comment by advance512** [comment]
I am planning on reviving this project, perhaps with support from MuleSoft.

If anyone is interested in contributing or help maintaining this project, let me know. :)

**Comment by justjacksonn** [comment]
I am just reading up on Go, so I doubt I can do much to help at this time,
but I am definitely interested in this. In particular I am hoping it is a
native RAML 1.0 parser for Go. I am also curious how the support of RAML
Types, JSON Schema and possibly even XML Schema is handled within Go. In
Java, for example, plain objects with JSON annotations are created, usually
one class for each JSON schema provided. How does Go handle this sort of
thing? Is it a single class with all the structures defined in it, or
individual classes under a specific package or what?

**Comment by advance512** [comment]
Have a look, here's an example project that implements JSON Schema in Go:
https://github.com/xeipuuv/gojsonschema

As of right now, this project only supports RAML 0.8, support for 1.0 is planned.

**Comment by sichvoge** [comment]
Is this project still alive?

**Comment by advance512** [comment]
Yes. The project has recently gained 2 new maintainers, which will become active in the next few weeks.

**Comment by sichvoge** [comment]
Have a look at https://github.com/raml-org/raml-tck. Might help for any parser test validation.

**Comment by gernest** [comment]
@advance512 anything in particular that you would like a helping hand on?

I wanted to implement raml , just stumbled on this. I'm quite familiar with go and parsing in general, but I would like to do something that is needed while getting familiar with this.

**Comment by advance512** [comment]
@gernest email me at diamant.alon@gmail.com

**Comment by gernest** [comment]
@advance512 why do we need email for this?

You can as well open an issue, describe it then mention me on it. Coz I can't promise to have much time dedicated to this one thing though.

I find it to be more convenient if raml was built on top of  the yaml library, I mean forking it then implement the raml logic while having direct access to the parser,  which is much better than my previous attempt to write everything from scratch.

A fast check on v1 it will non trivial to add all the features and being robust in time of speed and efficiency.

I might be wrong on this, since I came across this library just a couple of hrs ago.

**Comment by advance512** [comment]
I want to add you to a discussion that the new maintainers are having, that's why. You don't have to, of course.

If you think you can contribute, PRs are indeed welcome.

**Comment by sichvoge** [comment]
@advance512 can you add me to any discussions? I might be able to help on the RAML workgroup side.

Also, are you guys talking to https://github.com/Jumpscale/go-raml/issues/162 as well? Is that a separate project?

**Comment by advance512** [comment]
Jumpscale's go-raml is an awesome project which does tons of things, like server/client code generation. Its scope is much broader than the scope of `go-raml/raml`. It includes a RAML parser that is based on `go-raml/raml`. 

@sichvoge - email me at diamant.alon@gmail.com

**Comment by pedroalvesbatista** [comment]
> I am planning on reviving this project, perhaps with support from MuleSoft.
> 
> If anyone is interested in contributing or help maintaining this project, let me know. :)

What about that support status ?
